firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -2969,7 +2969,7 @@ extension String {
     public static let SettingsOfferClipboardBarStatus = MZLocalizedString(
         key: "Settings.OfferClipboardBar.Status",
         tableName: nil,
-        value: "When Opening Firefox",
+        value: "When opening Firefox",
         comment: "Description displayed under the ”Offer to Open Copied Link” option. See https://bug1223660.bmoattachments.org/attachment.cgi?id=8898349")
 }
```

Comment:
Everytime a string is changed we need to update the key
`key: "Settings.OfferClipboardBar.Status",` will become key: "Settings.OfferClipboardBar.StatusV2",